ui/StatusQ/src/StatusQ/Controls/StatusTagItem.qml

```diff
@@ -69,6 +69,8 @@ Control {
 
     implicitHeight: 30
     horizontalPadding: d.tagMargins
+    font.pixelSize: 15
+    font.bold: false
```

Comment:
But you might want to setup the correct `font.family` so that you can pass it down to the `StatusBaseText` as a whole